WORKSPACE

```diff
@@ -5,9 +5,6 @@ load("//:package.bzl", "rules_sass_dependencies", "rules_sass_dev_dependencies")
 rules_sass_dependencies()
 rules_sass_dev_dependencies()
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
```

Comment:
`node_repositories()` is no longer explicitly since `sass_repositories()` calls `yarn_install` which will call `node_repositories()` if it has not already been called.